std_msgs/String duckie_id
std_msgs/String[] in_fov
std_msgs/Bool on_service
std_msgs/Int8 collision_level
geometry_msgs/Pose2D pose
geometry_msgs/Twist velocity
